android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageView.kt

```diff
@@ -447,6 +497,20 @@ private val storeDateFormatISO: DateTimeFormatter = DateTimeFormatter.ISO_LOCAL_
 @OptIn(ExperimentalMaterial3Api::class)
 private val displayDateFormatter = DatePickerDefaults.dateFormatter(selectedDateSkeleton = "MM/dd/yyyy")
 
+fun canCreate(uiState: CreateListingPageUiState.Loaded) : Boolean {
+	return !adressIsEmpty(uiState.address) &&
+		uiState.price != 0 &&
+		uiState.startDate != "" &&
+		uiState.endDate != "" &&
+		uiState.numBedrooms != 0
```

Comment:
You can probably just check to make sure that `price` and `numBedrooms` is positive. Just in case they try to input a negative number.

Reply:
done